listenbrainz/webserver/views/test/test_index.py

```diff
@@ -26,7 +26,8 @@ def test_index(self):
 
     def test_downloads(self):
         resp = self.client.get(url_for('index.downloads'))
-        self.assert_redirects(resp, url_for('index.data'))
+        self.assertEqual(resp.status_code, 302)
+        self.assertEqual(resp.location, url_for('index.data'))
```

Comment:
we didn't do it in CB, but I realised that we have a base test class (ServerTestCase) where we could implement our own assertRedirects that checks both the status code and the location without the full protocol/hostname. Do you think it's worth it? Note that we have a "variant" of this test also in https://github.com/metabrainz/listenbrainz-server/blob/981e2fe6413e344e78f255a2d605fd5045d5fb11/listenbrainz/webserver/views/test/test_user.py#L48-L57

Reply:
Makes sense, added